yarn/src/main/scala/org/apache/spark/deploy/yarn/ApplicationMaster.scala

```diff
@@ -568,12 +568,16 @@ private[spark] class ApplicationMaster(
     }
 
     override def receiveAndReply(context: RpcCallContext): PartialFunction[Any, Unit] = {
-      case RequestExecutors(requestedTotal, localityAwareTasks, hostToLocalTaskCount) =>
+      case RequestExecutors(
+          requestedTotal,
+          localityAwareTasks,
+          hostToLocalTaskCount,
+          nodeBlacklist) =>
```

Comment:
same here for each arg on its own line -- no big deal, but IMO I'd put them on the same line